README.md

```diff
@@ -16,7 +16,7 @@ To learn how to contribute to the MetaMask project itself, visit our [Internal D
     - If you are using [nvm](https://github.com/creationix/nvm#installation) (recommended) running `nvm use` will automatically choose the right node version for you.
 - Install [Yarn](https://yarnpkg.com/en/docs/install)
 - Install dependencies: `yarn`
-- Copy the `.metamaskrc.dist` file to `.metamaskrc`
+- Copy the `.metamaskrc.dist` file to `.metamaskrc`, and replace the `INFURA_PROJECT_ID` value with your own personal [Infura Project ID](https://infura.io/docs).
```

Comment:
We might have a few extra things to suggest people. Maybe this should be a list?:

```suggestion
- Copy the `.metamaskrc.dist` file to `.metamaskrc`
    - Replace the `INFURA_PROJECT_ID` value with your own personal [Infura Project ID](https://infura.io/docs).
```

Reply:
Good point, that makes sense